Sustainsys.Saml2/WebSSO/Saml2RedirectBinding.cs

```diff
@@ -55,12 +55,13 @@ private static string AddSignature(string queryString, ISaml2Message message)
             string signingAlgorithmUrl = message.SigningAlgorithm;
 
             queryString += "&SigAlg=" + Uri.EscapeDataString(signingAlgorithmUrl);
-            var signatureDescription = (SignatureDescription)CryptoConfig.CreateFromName(signingAlgorithmUrl);
+            var signatureDescription = (SignatureDescription)CryptographyExtensions.CreateAlgorithmFromName(signingAlgorithmUrl);
             HashAlgorithm hashAlg = signatureDescription.CreateDigest();
             hashAlg.ComputeHash(Encoding.UTF8.GetBytes(queryString));
             AsymmetricSignatureFormatter asymmetricSignatureFormatter = 
                 signatureDescription.CreateFormatter(
-                    ((RSACryptoServiceProvider)message.SigningCertificate.PrivateKey)
+					EnvironmentHelpers.IsNetCore ? message.SigningCertificate.PrivateKey :
```

Comment:
IANA crypto expert, but according to this Stack Overflow discussion I believe `certificate.GetRSAPrivateKey()` is the future-proof way of doing this: https://stackoverflow.com/a/49777672/3191599

Reply:
@nbarbettini Thanks for pointing this out. I've still not had time to review this PR (stuck in a full time project), so help like this is very valuable.

Reply:
Thanks, I put fixing that on my todo list.